ipad.css

```diff
@@ -0,0 +1,221 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
your main.css file is your base css file. when the screen is smaller than 800px, the styling from your ipad media file will be "included". this means that you don't need to copy over all your css code from main.css. all the styling from that file will carry over. code from ipad css file will override that of the main.css file if there is a conflict.